src/installer/installer.go

```diff
@@ -425,40 +424,28 @@ func (i *installer) workerWaitFor2ReadyMasters(ctx context.Context) error {
 
 	i.log.Info("Waiting for 2 ready masters")
 	i.UpdateHostInstallProgress(models.HostStageWaitingForControlPlane, "")
-	start := time.Now()
-	err := utils.WaitForPredicate(waitForMastersTimeout, generalWaitInterval, func() bool {
-		var callErr error
-		cluster, callErr = i.inventoryClient.GetCluster(ctx, false)
-		if callErr != nil {
-			i.log.WithError(callErr).Errorf("Getting cluster %s", i.ClusterID)
-			return false
+	_ = utils.WaitForPredicate(waitForeverTimeout, generalWaitInterval, func() bool {
```

Comment:
We need to bail out if the cluster or host is in error state.  Otherwise, it will loop forever

Reply:
we don't, service will send installation stop command in case cluster is in error

Reply:
/lgtm